compact.go

```diff
@@ -482,6 +480,14 @@ func (c *LeveledCompactor) write(dest string, meta *BlockMeta, blocks ...BlockRe
 	if err := renameFile(tmp, dir); err != nil {
 		return errors.Wrap(err, "rename block dir")
 	}
+
+	level.Info(c.logger).Log(
```

Comment:
IMO we need to log on successful compaction only, right?

Reply:
Hmm, if it fails, it is good to know which blocks were involved. Is it worth printing out the blocks being compacted as well? And print out before the compaction?

Reply:
Yea, my worry is that we log this and assumed everything is fine and the error is for some other compaction.. what do you think?

Shouldn't we make sure our `errors.Wrap` have all required info (what blocks are involved) in case of error?

I will end up passing here compacted block metas as well for this, but maybe that's ok.

Reply:
Fair enough, yep :) Need to fix our errors.